### PR TITLE
Small update to required permissions for reading app role assignments

### PR DESCRIPTION
**Pull request by psignoret**

### CODE REVIEW

File: api-reference/beta/api/serviceprincipal-list-approleassignedto.md [hunk 1]
```diff
@@ -25,9 +25,9 @@ One of the following permissions is required to call this API. To learn more, in
 
 |Permission type      | Permissions (from least to most privileged)              |
 |:--------------------|:---------------------------------------------------------|
-|Delegated (work or school account) | Directory.Read.All, AppRoleAssignment.ReadWrite.All, Directory.ReadWrite.All, Directory.AccessAsUser.All  |
+|Delegated (work or school account) | Application.Read.All, Directory.Read.All, Application.ReadWrite.All, AppRoleAssignment.ReadWrite.All, Directory.ReadWrite.All, Directory.AccessAsUser.All  |
```
Comment (jirizacal) [inline api-reference/beta/api/serviceprincipal-list-approleassignedto.md:28]: I think we should also remove the AppRoleAssignment.ReadWrite.All. That scope does not grant any explicit read permissions really. Read of appRoleAssignments is included in servicePrincipals/read today. So you need at least Application.Read.All anyway.

File: api-reference/beta/api/serviceprincipal-list-approleassignedto.md [hunk 2]
```diff
@@ -25,9 +25,9 @@ One of the following permissions is required to call this API. To learn more, in
...
 |Delegated (personal Microsoft account) | Not supported.    |
-|Application | Directory.Read.All, AppRoleAssignment.ReadWrite.All, Directory.ReadWrite.All |
+|Application | Application.Read.All, Directory.Read.All, Application.ReadWrite.All, AppRoleAssignment.ReadWrite.All, Directory.ReadWrite.All |
```
Comment (jirizacal) [inline api-reference/beta/api/serviceprincipal-list-approleassignedto.md:30]: Same here and the other places too.

File: api-reference/beta/api/serviceprincipal-list-approleassignedto.md (hunk 1 above)
Comment (psignoret) [reply api-reference/beta/api/serviceprincipal-list-approleassignedto.md]: Thanks! Updated.

File: api-reference/beta/api/serviceprincipal-list-approleassignedto.md (hunk 2 above)
Comment (psignoret) [reply api-reference/beta/api/serviceprincipal-list-approleassignedto.md]: Updated.

**Pull request merged**